packages/fluentui/docs/src/examples/components/Breadcrumb/Types/BreadcrumbExampleIconDivider.tsx

```diff
@@ -0,0 +1,21 @@
+import * as React from 'react';
+import { Breadcrumb } from '@fluentui/react-northstar';
+import { ChevronEndMediumIcon } from '@fluentui/react-icons-northstar';
+
+const BreadcrumbExampleIconDivider = props => (
+  <Breadcrumb aria-label="breadcrumb">
+    <Breadcrumb.Item>Location</Breadcrumb.Item>
+    <Breadcrumb.Divider content=":" />
```

Comment:
IMO, "Location" is not a `Breadcrumb` item in this case

```suggestion
```

Reply:
I checked this: https://github.com/ant-design/ant-design/blob/master/components/breadcrumb/demo/separator-indepent.md
Should I just drop the `location:` part?

Reply:
Going to drop it according to phone discussion